db/migrate/20150317210343_convert_target_minutes_to_hours.rb

```diff
@@ -0,0 +1,13 @@
+class ConvertTargetMinutesToHours < ActiveRecord::Migration
+  class TempTarget < ActiveRecord::Base
+    self.table_name = 'targets'
+  end
```

Comment:
Yes, an up and a down.

Otherwise, as far as I know, except for add/remove columns and such, the same code of the change fuction will be run and this is not exactly what we ideally need (we would need to get the previous value back). Makes sense?